src/main/java/org/osc/controller/nsfc/api/NeutronSfcSdnRedirectionApi.java

```diff
@@ -285,8 +341,19 @@ public NetworkElement updateNetworkElement(NetworkElement serviceFunctionChain,
     @Override
     public void deleteNetworkElement(NetworkElement serviceFunctionChain) throws Exception {
 
+        // NEW //////////////////////////////////////////
         this.utils.throwExceptionIfNullElementAndId(serviceFunctionChain, "Service Function Chain Id");
+        PortChain portChain = this.osClient.sfc().portchains().get(serviceFunctionChain.getElementId());
```

Comment:
do we need a get before remove? we should just be able to delete and handle any exception.

also for 404...we should ignore since it means its already been deleted